docs/src/development/transfer-dedicated.md

```diff
@@ -1,149 +1,89 @@
-However, those are only useful for a full restore of the environment and can only be done by the Platform.sh team.
-At times, you'll want to make a manual backup yourself.
+## Back up your files
 
-To create a manual ad-hoc backup of all files on the staging or production environment, use the standard `rsync` command.
+Platform.sh automatically creates backups of the Staging and Production environments of a Dedicated cluster every six hours.
+These are only useful to fully restore an environment and are managed by the support team.
 
-```bash
-rsync -avzP <USERNAME>@<CLUSTER_NAME>.ent.platform.sh:pub/static/ pub/static/
-```
-
-That will copy all files from the `pub/static` directory on the production instance to the `pub/static` directory,
-relative to your local directory where you're running that command.
-
-## Backing up the staging and production database
-
-To backup your database to your local system, you need to get the database credentials to use.
-
-First, login to the cluster and run the following command:
+You can make a manual local backup yourself by downloading data from your environment to your local system by running the following command:
 
 ```bash
-echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
+platform scp --project <PROJECT_ID> --environment <ENVIRONMENT> -r remote:<DIRECTORY_TO_SYNCHRONIZE> <LOCAL_DIRECTORY>
 ```
 
-Which should give a JSON output containing something like this:
-
-```json
-"database" : [
-      {
-         "path" : "main",
-         "service" : "mysqldb",
-         "rel" : "mysql",
-         "host" : "database.internal",
-         "ip" : "246.0.80.64",
-         "scheme" : "mysql",
-         "cluster" : "jyu7wavyy6n6q-main-7rqtwti",
-         "username" : "user",
-         "password" : "",
-         "query" : {
-            "is_master" : true
-         },
-         "port" : 3306
-   }
-]
-```
+This command copies all files from the `<DIRECTORY_TO_SYNCHRONIZE>` in the environment you want to backup
+to your `<LOCAL_DIRECTORY>`.
+Before running the command, make sure that you don't overwrite local data (or do a backup first).
 
-The part you want is the user, password, and "path", which means the DB name.
-Ignore the rest.
+## Back up your database
 
-Now, run the following command on your local computer:
+To backup your database, adapt and run the following command on your local computer:
 
 ```bash
-ssh <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysqldump --single-transaction -u <user> -p<pass> -h localhost <dbname> | gzip' > database.gz
+platform db:dump --gzip --project <PROJECT_ID> --environment <ENVIRONMENT> 
 ```
 
-That runs a `mysqldump` command on the server, compress it using `gzip`,
-and stream the output to a file named `database.gz` on your local computer.
+For more backup options and examples, see [MariaDB/MySQL](../add-services/mysql/_index.md#exporting-data).
```

Comment:
:lady_beetle: :arrow_forward: This is again using the page name as the link as opposed to part of the sentence. It's better to have a sentence that includes the information that the reader would want
```suggestion
For more backup options and examples, see how to [export data from an SQL database](../add-services/mysql/_index.md#exporting-data).
```